src/test/java/org/apache/commons/lang3/time/DateUtilsTest.java

```diff
@@ -693,6 +693,59 @@ public void testToCalendar() {
             // expected
         }
     }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNotNull() {
+        assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the Date back", date1, DateUtils.toCalendar(date1, zone).getTime());
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNull() {
+        try {
```

Comment:
This is JUnit 4 feature.  I've fixed that but note that we cannot use JUnit 3 anymore.